test/e2e/scheduling/ubernetes_lite_volumes.go

```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
-var _ = Describe("[sig-storage] Multi-AZ Clusters", func() {
+var _ = SIGDescribe("Multi-AZ Cluster Volumes", func() {
```

Comment:
Why is this test related to scheduling?

Reply:
The original code mod to enable this feature was in the scheduler.  The key is in the check below - "should schedule pods in the same zones as statically provisioned PVs"

Reply:
He is moving the ubernetes-lite tests to a proper home.

Reply:
The fact that the test has "schedule pods" in the comment is not enough IMO. The question is "does this test mainly tests a scheduling functionality or the functionality of a different component?". I don't see how its purpose is to test a functionality of scheduler.

Reply:
It's validating this predicate - https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/algorithm/predicates/predicates.go#L483

Reply:
But TBH this test kind of crosses over both storage and scheduling... there isn't really a good home for such things.

Reply:
@marun - In chatting with @bsalamat I think we want storage to own these tests.  Even though they are validating predicate behavior, most of the code changes were made by that sig and any issue triage and maintenance should probably be on them.  /cc @kubernetes/sig-storage-misc

Reply:
As the creator of these tests, I am totally fine with owning it.  It's testing the persistent volume admission controller + volume zonal scheduler predicate, and not so much a generic scheduling functionality.

Reply:
Would you like to raise a PR to move it to test/e2e/storage or was adding ``[sig-storage]`` back to the description in #58841 sufficient?

Reply:
I think adding it back to the description is fine for now.  I believe this test does use some common functions in the main ubernetes_lite.go file which is why they were put in the same directory.  It'll take some refactoring out of the common parts to put it into the storage directory.